### ISSUE TITLE
Unit Test Naming Conventions

### ISSUE THREAD

**Issue by mgmccarthy** [HTBox/allReady#1198]
I vote:
- handler tests end in "Should"
- controller tests end in "Tests". the reason I think this is b/c some of our controller's are so large, and some of the semantics are "long" b/c of the work they're doing in a single action method, that ending in the word "Tests", IMO, makes naming the unit tests easier
- anything that is NOT a controller unit test (providers, services, etc..) should end "Should"

thoughts?

**Comment by stevejgordon** [comment]
@mgmccarthy Sounds reasonable to me. Probably something we introduce over time. Perhaps some good jump in issues to on-board new contributors as well. Something for our coding conventions concept as well.

**Comment by mgmccarthy** [comment]
@stevejgordon the code is already moving in this direction, but I agree, this refactoring is a good "jump-in" label... 

@tonysurma did yo want to mark this "jump-in" as @stevejgordon suggested?

**Comment by tonysurma** [comment]
Sounds like a good item for folks to jump in on so agree.
Also sounds like an item to add to wiki
@MisterJames any other thoughts?

**Comment by bmaluijb** [comment]
@tonysurma Love to pick this up. can you assign this to me please?

**Comment by mgmccarthy** [comment]
@bmaluijb, what would also be nice to roll into this task is not only the technical work, but to update our "Humanitarian Toolbox Developer Guide" at http://htbox.github.io/ under the DevelopmentGuidelines section at http://htbox.github.io/development-guidelines.html.

This site is a separate repository that is written using Markdown. You can find the repo here:
https://github.com/HTBox/htbox.github.io

If that's asking too much, @tonysurma, I recommend breaking out the updating of the Developer Guide into a separate Issue and labeling with jump-in.

**Comment by bmaluijb** [comment]
Sure, that seems a good fit. It seems silly to do the technical work and keep the documentation outdated.

**Comment by bmaluijb** [comment]
@tonysurma could you please assign me to this one?

**Comment by stevejgordon** [comment]
@tonysurma I was trying to assign @bmaluijb but seems he's not in the list. Over to you!

**Comment by tonysurma** [comment]
Yeah with a recent update to GitHub you have to invite someone to repo before you can assign.  Will do that when I am on a PC later.  Thanks

**Comment by bmaluijb** [comment]
@tonysurma Will this interfere with our code-a-thon that is coming up? Can we turn it off temporarily for that day or have people work on issues without being assigned to them?

**Comment by bmaluijb** [comment]
@tonysurma any news on this one?

**Comment by tonysurma** [comment]
@bmaluijb sorry for delay, assigning and related items are the one thing I can't do on mobile and been in transit lately.

As for the codeathon, no this won't.  This has been in place by GitHub for about a year and isn't a turn on/off item.  What folks do is put a comment on the item they are working on to say "I am working on this" or similar and that is the flag for others.

**Comment by tonysurma** [comment]
@bmaluijb you have been invited to collaborate on this repository, once you accept I can assign you

**Comment by bmaluijb** [comment]
Thanks @tonysurma !

**Comment by bmaluijb** [comment]
@tonysurma I've created two PR's for this:
- https://github.com/HTBox/allReady/pull/1404
- https://github.com/HTBox/htbox.github.io/pull/6

**Comment by marigerr** [comment]
Hi, I'm new to the project and going through 'jump-in' issues to hopefully find issues to contribute on.  It seems that this issue has been resolved with merged PRs from @bmaluijb and can be closed?
